Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -226,11 +226,17 @@ The dashboards have been created by [@galexrt](https://github.com/galexrt). For
     The dashboards are only compatible with Grafana 7.2.0 or higher.
     Also note that the dashboards are updated from time to time, to fix issues and improve them.
 
-The following Grafana dashboards are available:
+The following Grafana dashboards are available via Grafana.com:
 
-- [Ceph - Cluster](https://grafana.com/grafana/dashboards/2842)
-- [Ceph - OSD (Single)](https://grafana.com/grafana/dashboards/5336)
-- [Ceph - Pools](https://grafana.com/grafana/dashboards/5342)
+- [Ceph - Cluster (ID: 2842)](https://grafana.com/grafana/dashboards/2842)
+- [Ceph - OSD (Single) (ID: 5336)](https://grafana.com/grafana/dashboards/5336)
+- [Ceph - Pools (ID: 5342)](https://grafana.com/grafana/dashboards/5342)
+
+and as JSON files:
```

Comment:
Instead of mentioning the json files here, what if we add a readme in the folder of the json files so contributors can understand how to update them in rook and on the grafana site? Or why would we link the json files here for the admins?

Reply:
Admins might want to directly access the dashboard JSON files so they don't need to use grafana.com/github.com directly from their Grafana instance/deployment method.

I have changed the section to point to the `dashboards/` folder and added a README file to the folder.